src/main/webapp/jsp/adminActivityLog.jsp

```diff
@@ -6,34 +6,34 @@
 <%@ taglib tagdir="/WEB-INF/tags/admin/activity" prefix="activity" %>
 
 <c:set var="jsIncludes">
-    <script type="text/javascript" src="<%= FrontEndLibrary.JQUERY_HIGHLIGHT %>"></script>
-    <script type="text/javascript" src="/js/adminActivityLog.js"></script>
+  <script type="text/javascript" src="<%= FrontEndLibrary.JQUERY_HIGHLIGHT %>"></script>
+  <script type="text/javascript" src="/js/adminActivityLog.js"></script>
 </c:set>
 
 <ta:adminPage bodyTitle="Admin Activity Log" pageTitle="TEAMMATES - Administrator" jsIncludes="${jsIncludes}">
-    <activity:filterPanel excludedLogRequestURIs="${data.excludedLogRequestUris}" actionListAsHtml="${data.actionListAsHtml}"
-                            shouldShowAllLogs="${data.shouldShowAllLogs}" shouldShowTestData="${data.shouldShowTestData}" filterQuery="${data.filterQuery}"
-                            queryKeywordsForInfo="${data.queryKeywordsForInfo}"/>
+  <activity:filterPanel excludedLogRequestURIs="${data.excludedLogRequestUris}" actionListAsHtml="${data.actionListAsHtml}"
+              shouldShowAllLogs="${data.shouldShowAllLogs}" shouldShowTestData="${data.shouldShowTestData}" filterQuery="${data.filterQuery}"
+              queryKeywordsForInfo="${data.queryKeywordsForInfo}"/>
```

Comment:
We need to decide how much indentation to use and standardise it. Here it's 12 spaces, but there are other places where it's not consistent.